src/app/store/ui/ui.ts

```diff
@@ -3,21 +3,12 @@ import { atomFamily } from 'jotai/utils';
 
 type LoadingState = 'idle' | 'loading';
 
+// TODO investigate jotai removal as we are close now
```

Comment:
I feel like some of these comments aren't necessary unless they related to some specific implementation concern. 

Yes, we'd like to remove jotai but I don't think a loose comment floating in the code help us get to a result, and to me it seems quite unkept to have so many TODOs in the code.

Can we make an issue to remove it instead? Bring this up in our engineering syncs. Rally the rest of the team to get it done.

Reply:
Thanks Kyran, I can open a task for it. There are a lot of rogue comments to remove here still so apologies for that. 

My intention was to remove them once final polish is applied on re-review of everything. 

I definitely don't want to litter the code with comments but I also don't want to forget ideas I had during this refactor. 

Here's a task - https://github.com/leather-wallet/extension/issues/4966 

It could be prudent to consider all of our state management needs now, taking into account we will begin work on other apps and platforms so we could try and align them.